### ISSUE TITLE
https://github.com/stavro/arc/wiki/Direct-to-S3-Uploads - WIKI outofdate

### ISSUE THREAD

**Issue by netflakes** [stavro/arc#239]
Hi there the documentation in the WIKI for client uploads is outdated

s3_key = "uploads/b1cfef0d-a83f-473f-960d-c37f9de2f777"
Arc.Storage.S3.url(s3_key) #=> generates a public url
Arc.Storage.S3.url(s3_key, signed: true) #=> generates a private, signed url

These functions don't have a single parameter either 3 or 4. I have built an S3 based client side upload with using ex_aws to build pre-signed URL for the upload and it works as expected - am using React for the upload.. and channels to retrieve the signed_url and then post the updated info back to phoenix once the upload is done. I have a `key` column/field on the video table - but am not sure how to use Arc.Storage.S3 to generate the correct URLS for clients to access the vidoes (public/private etc) - Can you please advise or update the Wiki - that would be awesome.

Steve

**Comment by pedep** [comment]
Duplicate of #229
